commands/move.js

```diff
@@ -120,6 +120,12 @@ module.exports = {
       return;
     }
 
+    //general public cannot enter hidden rooms
+    if (exit.hidden && !socket.user.admin) {
```

Comment:
You can only enter a hidden room if you're an admin?

Reply:
This is so players cannot (un)wittingly enter a room that is currently hidden from them.